Prevent br normalization in ENTER_BR mode (#4895)
ckeditor/ckeditor4: tests/core/editable/manual/enterbr.md

@bender-tags: 4.16.3, bug, 3858
@bender-ui: collapsed
@bender-ckeditor-plugins: toolbar, wysiwygarea, clipboard, enterkey, format,

1. Open browser's console.
2. Copy two paragraphs placed above the editor. `<p>a</p><p>b</p>` and paste into editable.
3. Hit enter twice.
4. Type any ( only one ) letter and remove it using `backspace`.
5. Paste the the same two paragraphs from `step 1` ( don't change cursor position ).

> **Expected**

Reviewer: According to our [code style guide](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#31-manual-tests) there should be an empty lines after `Expected` and `Unexpected`. Please add them.

>  * The paragraphs from `step 5` are pasted correctly.

Reviewer: ```suggestion
 * The paragraphs from `step 5` look the same as in `step2`.
```

>  * The indentation is correct.

Reviewer: The expectations should be very clear. `Correct` therm sometimes could be interpreted differently by two different persons. I assume, that here, we need something like:`Pasted paragraphs are nor indented` or sth similar?

Also, we need additional mention that paragraphs pasted in step 5 are starting from the cursor position, so no lines were 'eaten' after the paste.

@bender-tags: 4.16.3, bug, 3858
@bender-ui: collapsed
@bender-ckeditor-plugins: toolbar, wysiwygarea, clipboard, enterkey, format, sourcearea

**Note**: Do not change the cursor position during the following steps.

1. Open browser's console.

> 2. Copy two paragraphs placed above the editor and paste then.

Reviewer: ```suggestion
2. Copy two paragraphs placed above the editor and paste them.
```

**Note**: This will also copy `<p>` elements that are around `a` and `b` letters.

3. Hit enter twice.
4. Type any (only one) letter and remove it using `backspace`.

> 5. Don't change the cursor position. Paste again the same two paragraphs.

Reviewer: If we want to keep the reminder here, despite the one at the beginning, maybe we can change the from a little bit:
```suggestion
5. Remember: don't change the cursor position. Paste again the same two paragraphs.
```
